packages/react/src/components/Tooltip/Tooltip.module.css

```diff
@@ -5,3 +5,16 @@
   border-radius: var(--fds-border_radius-medium);
   font: var(--fds-typography-paragraph-xsmall);
 }
+
+.wrapper.inverted {
+  background: var(--fds-semantic-surface-neutral-subtle);
+  color: var(--fds-semantic-text-neutral-default);
```

Comment:
Should this not use the inverted token? `--fds-semantic-surface-on_inverted-default` and `--fds-semantic-text-neutral-on_inverted`?

Reply:
Updated to use `--fds-semantic-surface-neutral-inverted`. Looks like it's the same color, but nice to use the correct variable.

Reply:
Aight. `color` should also be `--fds-semantic-text-neutral-on_inverted` in `wrapper` then, instead of "white" 🙈 ?

Reply:
It matches with figma?
<img width="868" alt="image" src="https://github.com/digdir/designsystemet/assets/14057205/5c108a87-2918-4486-9378-b8cb8bd59dc2">